src/componenets/modal/AddAtFolder.tsx

```diff
@@ -1,12 +1,22 @@
+import { SyntheticEvent } from "react";
 import close from "../../images/_close.png";
 
-function AddAtFolder({ toggleModal }) {
+type AddAtFolderType = {
```

Comment:
-Type이라는 접미사는 지양하는게 좋습니다~
```suggestion
type AddAtFolderProps = {
```